mixer/cmd/root.go

```diff
@@ -105,6 +108,18 @@ var RootCmd = &cobra.Command{
 		// If so: inform, stage, and exit.
 		// If not: run command in container and cancel pre-run
 		if !cmdContains(cmd, "format-bump") && !cmdContains(cmd, "upstream-format") && cmdContains(cmd, "build") {
+			// --offline=true AND --native=false, try to see if container exists
+			if !networkCheck && !builder.Native {
+				fmt.Println("Warning: Unable to determine upstream format in --offline mode, build may fail if format boundary is crossed.")
```

Comment:
Nitpick: Would prefer phrasing like ".. may fail if building across format boundaries". As it it sounds like the build would fail only if it is a build intended to cross a boundary.